helm/resource_release_test.go

```diff
@@ -918,6 +971,29 @@ func TestGetValuesString(t *testing.T) {
 	}
 }
 
+func TestIsLocalChart(t *testing.T) {
+
+	type test struct {
+		chartPath     string
+		repositoryURL string
+		isLocalChart  bool
+	}
+
+	tests := []test{
+		{chartPath: "./testdata/charts/test-chart", repositoryURL: "", isLocalChart: true},
+		{chartPath: "", repositoryURL: "https://charts.bitnami.com/bitnami", isLocalChart: false},
+		{chartPath: "redis", repositoryURL: "https://charts.bitnami.com/bitnami", isLocalChart: false},
+	}
+
+	for i, tc := range tests {
+		if result := isLocalChart(tc.chartPath, tc.repositoryURL); result != tc.isLocalChart {
+			t.Fatalf("[%v] error in isLocalChart; expected %v, got %v", i, tc.isLocalChart, result)
```

Comment:
I was playing with this and the error output isn't very useful, I would change it to this:

```go
t.Fatalf("[%v] error in isLocalChart; expected isLocalChart(%q, %q) == %v, got %v", i, tc.chartPath, tc.repositoryURL, tc.isLocalChart, result)

```